lib/controllers/commands/submit_questionnaire_command.dart

```diff
@@ -1,8 +1,17 @@
+import 'package:get/get.dart';
+import 'package:prapare/models/fhir_questionnaire/survey/export.dart';
+
+import '../controllers.dart';
 import 'abstract_command.dart';
 
 class SubmitQuestionnaireCommand extends AbstractCommand {
   @override
   Future<void> execute() async {
-    //todo: submit questionnaire
+    final UserResponsesController responsesController = Get.find();
```

Comment:
Because this command extends AbstractCommand, responsesController and QuestionnaireController have already been instantiated. Deleting lines 10 - 11, since they're redundant.